lib/galaxy/jobs/runners/slurm.py

```diff
@@ -23,7 +23,21 @@ class SlurmJobRunner( DRMAAJobRunner ):
     restrict_job_name_length = False
 
     def _complete_terminal_job( self, ajs, drmaa_state, **kwargs ):
-        def __get_jobinfo():
+        """
+        Returns not None if job was not actually terminal.
```

Comment:
Returns `True` if not actually terminal. Returns `None` otherwise. Not sure that it makes sense but that's the way it currently is.